src/main/kotlin/app/revanced/cli/command/utility/InstallCommand.kt

```diff
@@ -44,20 +45,35 @@ internal object InstallCommand : Runnable {
                 }.install(Installer.Apk(apk, packageName))
             } catch (e: Exception) {
                 logger.severe(e.toString())
+                throw e
```

Comment:
Why are we logging when throwing the exception which will be logged again?

Reply:
> which will be logged again?

It won't. The err is rethrown only for the sake of control flow.

Reply:
Since this is a binary situation, you can use return logic.

Reply:
Think we should continue this discussion under the [return code chain](https://github.com/ReVanced/revanced-cli/pull/365#discussion_r2051780152)